tokenizers/src/tokenizer/mod.rs

```diff
@@ -100,6 +104,14 @@ pub trait PostProcessor {
         pair_encoding: Option<Encoding>,
         add_special_tokens: bool,
     ) -> Result<Encoding>;
+    /// Process method that is used when a processor is part of `processors::sequence::Sequence`
+    fn process_chain(
+        &self,
+        _encodings: Vec<Encoding>,
+        _add_special_tokens: bool,
+    ) -> Result<Vec<Encoding>> {
```

Comment:
@Narsil this is the preliminary signature we agreed upon. Please check its implementation & let me know if there is anything you wanna change